AK Zeus/app/src/main/java/ayds/zeus/songinfo/home/model/entities/SpotifySong.kt

```diff
@@ -17,20 +19,19 @@ data class SpotifySong(
   override val artistName: String,
   override val albumName: String,
   override val releaseDate: String,
+  override val releaseDatePrecision:String,
   override val spotifyUrl: String,
   override val imageUrl: String,
   override var isLocallyStoraged: Boolean = false
-) : Song {
-
-    val year: String = releaseDate.split("-").first()
-}
+) : Song {}
```

Comment:
Aca lo contrario, el : debe estar pegado al anterior 
```suggestion
): Song {}
```